src/main/resources/MAT.properties

```diff
@@ -36,6 +36,8 @@ mat.qdm.default.expansion.id=Most Recent Code System Versions in VSAC
 #At every 5th min
 mat.cache.expiry.time=300000
 
+mat.fhirMatServices.url=http://localhost:9090

```

Comment:
probably should be passed as a system property.
we pass 
-DFHIR_ORCH_MEASURE_SRVC_URL=https://matdev.semanticbits.com/mat-fhir-services/orchestration/measure

Reply:
And we will have to unify it soon..